dubbo-config/dubbo-config-spring/src/main/java/org/apache/dubbo/config/spring/util/AnnotationUtils.java

```diff
@@ -93,6 +92,14 @@ public static String resolveInterfaceName(Service service, Class<?> defaultInter
      * @throws IllegalStateException if interface name was not found
      */
     public static String resolveInterfaceName(AnnotationAttributes attributes, Class<?> defaultInterfaceClass) {
+        Boolean generic = getAttribute(attributes, "generic");
```

Comment:
looks like this is unnecessary, since it works only for `@Service`

Reply:
No, it also use for `@Reference`. I think the method's comments is wrong.

Reply:
nope, my mistake.